javaparser-core-serialization/src/main/java/com/github/javaparser/serialization/JavaParserJsonDeserializer.java

```diff
@@ -42,13 +43,33 @@
  * Deserializes the JSON file that was built by {@link JavaParserJsonSerializer}.
  */
 public class JavaParserJsonDeserializer {
-
+    public static final String COMMENT_PROPERTY_KEY =
+            CommentMetaModel.NAME.substring(0, 1).toLowerCase() + CommentMetaModel.NAME.substring(1);
+
+    /**
+     * Deserializes json, contained by JsonReader, into AST node.
+     * The root node and all its child nodes will be deserialized.
+     * @param reader json-p reader (object-level reader, <a href="https://javaee.github.io/jsonp/"></a>see their docs</a>)
```

Comment:
the a tag seems closes twice
```suggestion
     * @param reader json-p reader (object-level reader, <a href="https://javaee.github.io/jsonp/">see their docs</a>)
```